documentation/embeddable-interactive-filtering.md

```diff
@@ -8,61 +8,46 @@ Interactives on the page bypass this "answer" approach. They use the interactive
 
 The numbering of items in the activity boils down to:
 
-    activity.questions.index(item) + 1
+    activity.reportable_items.index(item) + 1
 
-This happens when individual 'answer' partials call the Embeddable::Answer#question_index to find the index of the 'question' on the page. The Xhtml partial  doesn't look for an index. Reportable interactives technically have an index, but they do not display it.
+This happens when individual 'answer' partials call the `Embeddable::Answer#question_index` to find the index of the 'question' on the page. The Xhtml partial  doesn't look for an index. Reportable interactives technically have an index because they are part of reportable_items, but they do not display it.
```

Comment:
Yes I'm pretty sure that Dan accepted this. 
Also interactives are not interleaved with questions. So the skips in numbers will only happen between pages.